frontend/src/sass/pages/main-landing.scss

```diff
@@ -116,7 +116,7 @@
   }
 }
 
-#help-pick-background{
```

Comment:
small issue but if you want to address it for code cleanliness / consistency there should be a space before the `{`